docs/en/build/linux.md

```diff
@@ -445,9 +453,11 @@ You can also activate other engines after the model.
 
 #### Build Demo
 
+Take ncnn backend as example:
+
 ```Bash
 cd ${MMDEPLOY_DIR}/build/install/example
 mkdir -p build && cd build
-cmake .. -DMMDeploy_DIR=${MMDEPLOY_DIR}/build/install/lib/cmake/MMDeploy
+cmake .. -DMMDeploy_DIR=${MMDEPLOY_DIR}/build/install/lib/cmake/MMDeploy -Dncnn_DIR=${NCNN_DIR}/build/install/lib/cmake/ncnn
```

Comment:
Is it necessary to add ncnn library when we use the built SDK? ncnn should have been linked into MMDeploy SDK when we build it. Add ncnn lib to `LD_LIBRARY_PATH` should be enough(or if it is not, we should make it.)

Reply:
done.